k8s-resources/mysql.yaml

```
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: dsmhack-2025-cultivate
+  name: shopeatlocal-mysql
+spec:
+  ports:
+    - port: 3306
+  selector:
+    app: mysql
+  clusterIP: None
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: dsmhack-2025-cultivate
+  name: mysql
+spec:
+  selector:
+    matchLabels:
+      app: mysql
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: mysql
+    spec:
+      imagePullSecrets:
+        - name: dockerconfigjson-github-com
+      initContainers:
+        - name: init
+          image: migrations-image
+          volumeMounts:
+            - mountPath: /docker-entrypoint-initdb.d
+              name: dump
+      containers:
+        - image: mysql:9
+          name: mysql
+          env:
+            - name: MYSQL_DATABASE
+              value: shopeatlocal
+            - name: MYSQL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: password
+                  name: mysql-secrets
+            - name: MYSQL_USER
+              valueFrom:
+                secretKeyRef:
+                  key: username
+                  name: mysql-secrets
+            - name: MYSQL_RANDOM_ROOT_PASSWORD
+              value: "true"
+          ports:
+            - containerPort: 3306
+              name: mysql
+          volumeMounts:
+#            - name: mysql-persistent-storage
+#              mountPath: /var/lib/mysql
+            - mountPath: /docker-entrypoint-initdb.d
+              name: dump
+          resources:
+            limits:
+              memory: 1024Mi
+              cpu: 1000m
+            requests:
+              memory: 32Mi
+              cpu: 10m
```

I might bump this up
```suggestion
              memory: 512Mi
              cpu: 200m
```